### DISCUSSION TITLE
How to serve fonts from fontsource on webpack-dev-server?

### DISCUSSION THREAD

**Discussion by dfontenot** [webpack/webpack#16786] [Q&A]
Hello,

I am attempting to follow [this guide from Chakra UI](https://chakra-ui.com/community/recipes/using-fonts) for custom fonts, but the rendered fonts setting in Chrome is still showing the fallback font. I am using option 1 in the guide, and in addition to following the steps I have also added the asset loader and CSS loader to my webpack config:

```
      {
        test: /\.css$/i,
        use: ['css-loader'],
      },
      {
        test: /\.(woff|woff2|eot|ttf|otf)$/i,
        type: 'asset/resource',
      },
```

I also do not see any requests for the actual .woff files from the Chrome network inspector. Is there something I am missing, or something else I need to do to get this setup to work correctly? Thanks.

**Comment by alexander-akait** [comment]
Hello, sorry without code we can't help, maybe you don't import them in CSS, I don't know, pleae provide more information, thank you
